src/components/wallet/methods/depositAsset.ts

```diff
@@ -213,7 +180,7 @@ export default async function depositAsset(
       }
     }
   } catch (err) {
-    this.loading = { ...this.loading, deposit: false }
+    finish()
```

Comment:
Always use the finish function so we don't run into problems where we've set the wrong loading key to false, and end up with a non-stop spinner.